qutebrowser/config/configtypes.py

```diff
@@ -1392,6 +1392,15 @@ class NewInstanceOpenTarget(BaseType):
                                ('window', "Open in a new window."))
 
 
+class DownloadPath(BaseType):
```

Comment:
I think the name is a bit misleading. What about `DownloadPathSuggestion` or so? This'd be consistent with other setttings with their own types (e.g. `NewInstanceOpenTarget`) as well.